src/api/app/models/package_issue.rb

```diff
@@ -2,6 +2,11 @@ class PackageIssue < ApplicationRecord
   belongs_to :package
   belongs_to :issue
 
+  scope :of_open_issues, -> (owner_id) { joins(:issue).where(issues: { state: 'OPEN', owner_id: owner_id})}
```

Comment:
Hm, I don't like this name

```
PackageIssue.of_open_issues(id)
```
sounds strange, doesn't it?

Not sure if @Ana06 complained about moving it into one scope but maybe having two scopes is more readable
```
PackageIssue.open.of_owner(owner_id)
```

@Ana06 ^^